translator/translate/otel/pipeline/jmx/translator.go

```diff
@@ -107,6 +108,7 @@ func (t *translator) Translate(conf *confmap.Conf) (*common.ComponentTranslators
 
 	if envconfig.IsRunningInContainer() {
 		translators.Receivers.Set(otlp.NewTranslatorWithName(common.JmxKey))
+		translators.Processors.Set(metricstransformprocessor.NewTranslatorWithName(common.JmxKey))
```

Comment:
What about the non-container use-case ?

Reply:
We're only adding this processor for container environments because the JMX gatherer already supports aggregation for this metric for EC2.